govcd/api_vcd_test.go

```diff
@@ -13,6 +13,22 @@ import (
 	"testing"
 )
 
+const (
+	// Names for entities created by the tests
+	OrgCreateTest            = "CreateOrgTest"
+	OrgDeleteTest            = "DeleteOrgTest"
+	OrgUpdateTest            = "UpdateOrgTest"
+	CatalogCreateTest        = "CreateCatalogTest"
```

Comment:
One suggestion for improvement. The purpose of this ask is to enable better readability and easier troubleshooting.

Right now the order of the words is not consistent (e.g. OrgCreateTest vs. CreateOrgTest). Suggestion is to rename the variables and the values to better match the test functions themselves as in the example below.

Function:
`Test_CreateCatalog(check *C)`
Change from:
`CatalogCreateTest        = "CreateCatalogTest"`
Change to:
`TestCreateCatalog        = "TestCreateCatalog"`

Having "Test" at the beginning of entity names should also proof more clear when looking at the vCD instance on which the tests are running right now. You would instantly see that this is a test.